### PR TITLE
URL changes from HTTP to HTTPS

### PR DESCRIPTION
**Pull request by michaelehrlein**
The URL changes from HTTP to HTTPS
Response from old URL was "307 / Temporary Redirect" an "Service unavailable" Exception as result

### CODE REVIEW

File: tests/bootstrap.php [hunk 1]
```diff
@@ -58,29 +58,33 @@
...
         'engine' => 'File',
         'prefix' => 'default_',
-        'serialize' => true
-    ]
+        'serialize' => true,
+    ],
+]);
+
+Configure::write('Error', [
+    'errorLevel' => E_ALL & ~E_USER_DEPRECATED,
```
Comment (tersmitten) [inline tests/bootstrap.php]: Why is this needed?

**Comment by tersmitten** [comment]
Thanks for your contribution (so far)!

Although this plugin should still work I would advice you to use https://github.com/Oefenweb/cakephp-vat-number-check-webservice

File: tests/bootstrap.php (hunk 1 above)
Comment (michaelehrlein) [reply tests/bootstrap.php]: "broken" Buildchain: https://travis-ci.org/github/Oefenweb/cakephp-vat-number-check/jobs/680926238

Comment (michaelehrlein) [reply tests/bootstrap.php]: reverted this Commit

Comment (tersmitten) [reply tests/bootstrap.php]: I see. It shouldn't be hard to fix these errors

Comment (tersmitten) [reply tests/bootstrap.php]: See https://github.com/Oefenweb/cakephp-vat-number-check-webservice/blob/3.x/tests/bootstrap.php

Comment (tersmitten) [reply tests/bootstrap.php]: > It shouldn't be hard to fix these errors

It was a bit hard and a lot of work, but everything is nice a shiny again :-)
